routes/socket/user-events.js

```diff
@@ -1789,7 +1789,7 @@ module.exports.handleUpdatedRemakeGame = (passport, game, data, socket) => {
  */
 module.exports.handleAddNewGameChat = (socket, passport, data, game, modUserNames, editorUserNames, adminUserNames, addNewClaim) => {
 	// Authentication Assured in routes.js
-	if (!game || !game.general || !data.chat) return;
+	if (!game || !game.general || !data.chat || !data.chat.replace(/\s*(\*|~|_){2,4}\s*|\s*\*\s*/i, '')) return;
```

Comment:
While this would theoretically work, placing your check after `data.chat.trim()` in 1793 could greatly simplify this and remove the need for whitespace matching (which could cause more problems with users using different locale whitespaces). 

Also if done after the trim, you could simply use
```javascript
if (/^((\*|~|_){2,4}|\*)$/i.exec(data.chat))
    return;
```
Alternatively, you could trim directly in this `if`
```javascript
if (
     !game || 
     !game.general || 
     !data.chat || 
     /^((\*|~|_){2,4}|\*)$/i.exec(data.chat.trim())
    )
     return;
```

Reply:
Wait, doesn't RegExp.exec() only check for parts of a string matches the RegExp, not if the whole string matches?

Reply:
Doesn't RegExp.exec only check and return matches that it has found? It doesn't check if the whole string matches, right?

Reply:
Generally, yes - but `^` and `$` are the characters to search for the start and the end of the string respectively, allowing you to search the entire string for a match.

Reply:
Ah, ok. I’ll fix that in another commit.